style.css

```diff
@@ -49,7 +49,8 @@ body {
   left: 0;
   right: 0;
   bottom: 0;
-  background-color: red;
+  min-width:80px;
+  background-color: #007bff;
```

Comment:
Matching the body background means the button is invisible.

Reply:
@immattdavison  I have changed the button background to orange and slider to blue, to make it easily distinguishable.